server/hap/HapProcessCeilometer.cc

```diff
@@ -883,6 +883,33 @@ HatoholError HapProcessCeilometer::fetchItem(const MessagingContext &msgCtx,
 	return err;
 }
 
+HatoholError HapProcessCeilometer::fetchTrigger(const MessagingContext &msgCtx,
```

Comment:
Many lines in this method is similar to those in getAlarmList.
Let's remove duplicated lines.